jaworski_cellseg/label_counting_widget.py

```diff
@@ -18,23 +59,43 @@ def create_label_counting_widget(viewer: napari.Viewer):
             "label": "Total count",
             "enabled": False,
         },  # Make the textbox read-only
+        save_results={
+            "widget_type": "FileEdit",
+            "mode": "w",
+            "filter": "*.csv",
+        },  # Dynamic choices will be set later
     )
     def cell_counting_widget(
-        label_layer_name: str = None,
-        binary_layer_name: str = None,
-        total_count: str = "",
+        label_layer_name: Optional[str] = None,
+        binary_layer_name: Optional[str] = None,
+        total_count: Optional[str] = "",
+        save_results: Optional[str] = None,
```

Comment:
It seems like these two are no longer used.
```suggestion
```

Reply:
Im leaving them because the number of params in the function must match the params in the `@magicgui ` annotation.

Reply:
Consider adding a comment.

Reply:
Done